src/Sulu/Bundle/MediaBundle/Media/Exception/CacheNotFoundException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\Exception;
+
+/**
+ */
+class CacheNotFoundException extends MediaException
+{
+    /**
+     * @param string $alias
+     */
+    public function __construct($alias)
```

Comment:
Can you maybe also add a getter for the `$alias`? Might be useful for the developer using this exception.